avidemux/qt4/i18n/avidemux_ru.ts

```diff
@@ -12626,22 +12627,22 @@ Please delete the idx2 file and reopen.</source>
     <message>
         <location filename="../../../avidemux_plugins/ADM_videoFilters6/vaapiFilters/ADM_vidVaapiFilter.cpp" line="+67"/>
         <source>vaapi: Resize</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">vaapi: Изменить размер</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>vaapi: Resize image using vaapi.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">vaapi: Изменить размер изображения с помощью vaapi.</translation>
     </message>
     <message>
         <location line="+134"/>
         <source>Width :</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">Ширина :</translation>
```

Comment:
I thought it's a local rule, like for special symbols or something. You can correct it.